pkg/controller/machine.go

```diff
@@ -279,7 +279,6 @@ func (c *Controller) validateMachine(prov cloud.Provider, machine *machinev1alph
 }
 
 func (c *Controller) syncHandler(key string) error {
-	var providerInstance instance.Instance
```

Comment:
Nice, that caused some NPEs which is why I added a `== nil` check for it in `ensuereNodeOwnerRefAndConfigSource`

Reply:
Oh it just got moved :S